db/connection.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config({ path: './config/config.env' });
+
+
+const mongoose = require('mongoose');
+
+const connectDB = async () => {
+  try {
+    await mongoose.connect(process.env.MONGODB_URI, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
```

Comment:
Tych dwóch kluczy nie potrzebujesz, pewnie jak zobaczysz w terminalu to masz informacje że są już niepotrzebne, tam będzie taka krótka informacja na ten temat.